integrations/sklearn/setup.py

```diff
@@ -0,0 +1,37 @@
+import versioneer
```

Comment:
Looks like you forgot to modify setup.py in neptune-client.

Reply:
everything needed to be done is described in `integgrations/README.md`. We can create the pypi token after merging this commit, but there's still some things missing from that instructions

Reply:
added sklearn to setup.py